### ISSUE TITLE
Separate Uno.UI.Toolkit to its own package

### ISSUE THREAD

**Issue by Youssef1313** [unoplatform/uno#10933]
Currently, Uno.UI can be referenced in Windows builds, but only Uno.UI.Toolkit is in the package:

![image](https://user-images.githubusercontent.com/31348972/211509997-e0a0d8d9-1ba5-4dd2-82de-273035b44db1.png)

For UAP, the package *only* contains Uno.UI.Toolkit. The rest of the package isn't used.

I think it makes sense to separate Uno.UI.Toolkit to its own NuGet package. For Windows projects/builds, we will only include `Uno.UI.Toolkit` instead of the whole `Uno.UI` package.

This can result in faster restore when building UAP only.

That will be part of https://github.com/unoplatform/uno/issues/10302

**Comment by MartinZikmund** [comment]
Could be part of 5x breaking changes

**Comment by MartinZikmund** [comment]
Ideally we should also avoid naming the package `Uno.UI.Toolkit` to avoid confusion with Uno Toolkit.

**Comment by jeromelaban** [comment]
Indeed, renaming the assembly can cause more trouble than it fixes. Creating a package may be interesting, though dependencies may be tricky (as well as the upgrade path). Let's see how 5x goes first.

**Comment by MartinZikmund** [comment]
Still relevant

**Comment by MartinZikmund** [comment]
# Repro generator

This issue was classified as not reproducible with a code sample or test.

**Reason:** This is a feature request about NuGet packaging (separating Uno.UI.Toolkit into its own package). It does not describe a bug that can be reproduced with a runtime test or manual sample.
